### PR TITLE
Implemented ssh support

### PR DESCRIPTION
**Pull request by mathcrosp**

### CODE REVIEW

File: ndd_slurm.py [hunk 1]
```diff
@@ -138,9 +140,37 @@ def run_master(args):
...
+        cmd += ['-s', '{}:{}'.format(host, args.p)]
+    put_non_required_options(args, cmd)
+    return cmd
+
+def get_ssh_slave_cmds(args):
+    SSH = ['ssh', '-o', 'PasswordAuthentication=no']
+    slaves = ','.join(args.d)
+    slaves = slaves.split(',')
+    cmds = [(SSH + [slave] + get_ssh_slave_ndd_cmd(args, slave, slaves))\
+           for slave in slaves]
```
Comment (andrey-malets) [inline ndd_slurm.py]: Probably it should have one extra space before this string:
```
  [one, two
   three]
```

Comment (mathcrosp) [inline ndd_slurm.py]: @andrey-malets, done.

File: ndd_slurm.py [hunk 2]
```diff
@@ -138,9 +140,37 @@ def run_master(args):
...
+    source = args.s if index == 0 else slaves[index-1]
+    cmd += ['-r', '{}:{}'.format(source, args.p)]
+    if index != len(slaves) - 1:
+        cmd += ['-s', '{}:{}'.format(host, args.p)]
+    put_non_required_options(args, cmd)
+    return cmd
+
+def get_ssh_slave_cmds(args):
+    SSH = ['ssh', '-o', 'PasswordAuthentication=no']
+    slaves = ','.join(args.d)
```
Comment (andrey-malets) [inline ndd_slurm.py]: Looks like we can optimize away a couple of strings here :)

**Comment by andrey-malets** [comment]
Thanks!

**Pull request merged**